### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by acycliq** [manubb/Leaflet.PixiOverlay#67]
Hi there

Just to say thanks for this lib. I have used it to build this [viewer  ]( https://acycliq.github.io/ca1/). It shows a part of the mouse brain (hippocampus/CA1). The dots are genes and polygons are cells.

**Comment by manubb** [comment]
Hi. Do you have a new url for your demo?  (The link above is broken.)

**Comment by acycliq** [comment]
oh yes, just updated it! Thanks

**Comment by manubb** [comment]
May i add this url in the main README ?

**Comment by acycliq** [comment]
yes sure although my coding is not the best out there. It may help some ppl though and give  them ideas on how to use your amazing library
